spec/ig-oncocore-config.json

```diff
@@ -25,7 +25,7 @@
 		"strategy": "hybrid",
 		"target": ["oncocore", "shr.base.Entry"]
 	},
-    "publisher": "The HL7 Cancer Interoperability Group sponsored by Clinical Interoperability Council Work Group (CIC)",
+    "publisher": "The MITRE Corporation",
```

Comment:
This is updated in the wrong file.  We can keep it here for now, I'll move it to the right spot, and then delete this file to remove confusion moving forward.